autosubmit/job/job_list.py

```diff
@@ -1049,10 +1049,11 @@ def find_current_section(self, job_section, section, dic_jobs, distance, visited
                     return distance
         return distance
 
-    def _calculate_natural_dependencies(self, dic_jobs, job, dependency, date, member, chunk,
-            graph, dependencies_keys_without_special_chars, distances_of_current_section,
-            distances_of_current_section_members, key, dependencies_of_that_section, chunk_list,
-            date_list, member_list, special_dependencies, max_distance, problematic_dependencies):
+
+    def _calculate_natural_dependencies(self, dic_jobs, job, dependency, date, member, chunk, graph,
+                                distances_of_current_section, key, dependencies_of_that_section,
+                                chunk_list, date_list, member_list, special_dependencies,
+                                max_distance, problematic_dependencies):
```

Comment:
Was the indentation fixed with a linter? PyCharm now complains about this. Please take a look at what PyCharm is showing in this PR (if you are not seeing the warnings we need to either adjust your PyCharm settings, or others have to adjust their settings to match yours).

![image](https://github.com/user-attachments/assets/17da4651-2fae-4377-8861-f6617e844a86)